src/include/cephfs/libcephfs.h

```diff
@@ -2285,6 +2285,29 @@ int ceph_get_snap_info(struct ceph_mount_info *cmount,
  * @param snap_info snapshot info struct (fetched via call to ceph_get_snap_info()).
  */
 void ceph_free_snap_info_buffer(struct snap_info *snap_info);
+
+/**
+ * perf counters via libcephfs API.
+ */
+
+/**
+ * Get a json string of performance counters
+ *
+ * @param cmount the ceph mount handle to use.
+ * @param perf_dump buffer holding the perf dump
+ *
+ * Returns 0 success with the performance counters populated in the
+ * passed in perf_dump buffer.
+ */
+int ceph_get_perf_counters(struct ceph_mount_info *cmount, char **perf_dump);
+
+/**
+ * free allocated perf counter buffer
+ *
+ * @param allocated buffer via ceph_get_perf_counters()
+ */
+void ceph_free_perf_counters(char *perf_dump);
```

Comment:
We should just indicate the buffer should be freed via `free(3)` and not add this.

Reply:
Fair enough.